os/test/src-jvm/ExampleTests.scala

```diff
@@ -271,7 +271,7 @@ object ExampleTests extends TestSuite{
             .filter(_._2.length > 0)
             .filter(!_._1.segments.contains("src_managed"))
 
-      assert(filesWithTooLongLines.length == 0)
+      Predef.assert(filesWithTooLongLines.length == 0, filesWithTooLongLines)
```

Comment:
What's the motivation? Wasn't it utest `assert` berfore, which is printing the `filesWithTooLongLines` automatically in case of failure?

Reply:
It wasnt printing *which files* had issues, now it does.